src/main/java/org/mule/extension/db/internal/domain/connection/DefaultDbConnection.java

```diff
@@ -159,4 +190,113 @@ public boolean isTransactionActive() {
   public boolean supportsContentStreaming() {
     return true;
   }
+
+  public PreparedStatement prepareStatement(String sql) throws SQLException {
```

Comment:
do not use this method (use jdbcConnection.prepareStatement) or change scope to private.